File: demo-app/app/src/javaApp/java/com/launchkey/android/authenticator/demo/ui/BaseDemoView.java
package com.launchkey.android.authenticator.demo.ui;

import com.launchkey.android.authenticator.sdk.core.authentication_management.AuthenticatorManager;
import com.launchkey.android.authenticator.sdk.ui.AuthenticatorUIManager;

public interface BaseDemoView {
    AuthenticatorManager getAuthenticatorManager();
    AuthenticatorUIManager getAuthenticatorUIManager();
}

File: demo-app/app/src/kotlinApp/java/com/launchkey/android/authenticator/demo/ui/BaseDemoView.kt
package com.launchkey.android.authenticator.demo.ui

import com.launchkey.android.authenticator.sdk.core.authentication_management.AuthenticatorManager
import com.launchkey.android.authenticator.sdk.ui.AuthenticatorUIManager

interface BaseDemoView {
    val authenticatorManager: AuthenticatorManager
    val authenticatorUIManager: AuthenticatorUIManager
}